### PR TITLE
Shared components snapshot tests

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/components/shared/Button/__tests__/component.spec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Button from '../component';
+
+describe('Button component', () => {
```
Comment (plzen) [inline src/components/shared/Button/__tests__/component.spec.js]: I think also should be test with only required props to check if default values is set properly

File: src/components/shared/Input/__tests__/component.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Input } from '../component';
+
+describe('Input component', () => {
```
Comment (plzen) [inline src/components/shared/Input/__tests__/component.spec.js]: The same is here:
1. Only required props
2. All props including optional
3+. Conditions

File: src/components/shared/Textarea/__tests__/component.spec.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Textarea from '../component';
+
+describe('Textarea component', () => {
```
Comment (plzen) [inline src/components/shared/Textarea/__tests__/component.spec.js]: Add tests for default props

File: src/components/shared/Button/__tests__/component.spec.js (hunk 1 above)
Comment (eugene-chr) [reply src/components/shared/Button/__tests__/component.spec.js]: done

File: src/components/shared/Input/__tests__/component.spec.js (hunk 2 above)
Comment (eugene-chr) [reply src/components/shared/Input/__tests__/component.spec.js]: done

File: src/components/shared/Textarea/__tests__/component.spec.js (hunk 3 above)
Comment (eugene-chr) [reply src/components/shared/Textarea/__tests__/component.spec.js]: done

**Pull request merged**